### ISSUE TITLE
Put MoM file somewhere readable to non-root user

### ISSUE THREAD

**Issue by lebensterben** [clearlinux/swupd-client#1496]
Currently, the system caches the MoM file of official repo at `/var/tmp/swupd/Manifest.MoM`, which is readable to all users.
If it doesn't exist, there's another in `/var/lib/swupd/VERSION/Manifest.MoM`. But this one, is only readable to root user. Because its `/var/lib/swupd` is only readable to root user.

For 3rd-party repos, I cannot find a MoM file under readable directory. I do find them at `/var/lib/swupd/3rd-party/REPO_NAME/VERSION/Manifest.MoM`. But that's no readable.

The main drawback for this is it's difficult to write shell completion for 3rd-party repos. Although this is possible, but I prefer not to ask for root privilege for running a shell completion script.

**Comment by lebensterben** [comment]
relevant issue: https://github.com/clearlinux/swupd-client/issues/490

**Comment by otaviobp** [comment]
Yes, this is indeed a problem. We need to improve how we handle permissions in /var/lib/swupd/
